design-system-docs/Gemfile

```diff
@@ -20,6 +20,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gem "bridgetown", "~> 1.0"
 gem "htmlbeautifier"
+gem "nokogiri"
```

Comment:
This is already available as part of actiontext, I think, but adding it here makes it clear we're using it as an explicit dependency.